src/EE70C0.c

```diff
@@ -16,18 +16,36 @@ extern s32 D_80244630_EEABE0;
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+// Needs data migrated
+#ifdef NON_MATCHING
+ApiStatus func_80242260_EE8810(Evt *script, s32 isInitialCall) {
```

Comment:
Duplicate function of `ItemChoice_WaitForSelection` so it can be renamed to `EE70C0_ItemChoice_WaitForSelection`

`D_80244630_EEABE0` is gonna be `EE70C0_ItemChoice_SelectedItemID`
`D_8024462C_EEABDC` is gonna be `EE70C0_ItemChoice_HasSelectedItem`

Reply:
Generally speaking, aren't these gonna be deduped once data is migrated anyway @ethteck?

Reply:
yeah I wouldn't worry about the names too much here

Reply:
Fair, doesn't change anything really.
One could probably argue it'd be nice for readability and consistency sake, but it's gonna be snapped once data is migrated anyway 👀

Reply:
ok so don't rename this?

Reply:
yeah dw about it